.github/workflows/kvrocks.yaml

```diff
@@ -448,6 +451,16 @@ jobs:
           update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-11 100
           echo "NPROC=$(nproc)" >> $GITHUB_ENV
 
+      - name: Setup Debian
+        if: ${{ startsWith(matrix.image, 'debian') }}
+        run: |
+          apt update
+          apt install -y gcc g++ make wget git autoconf automake python3 python3-pip curl tar gzip cmake 
```

Comment:
I think, will be better: apt install -y build-essential wget cmake libtool python3 libssl-dev python3 python3-pip

Reply:
build-essential is a virtual package, including make and g++ and gcc.